src/Orders.java

```diff
@@ -0,0 +1,67 @@
+public class Orders {
+
+
+    int client_ID;
```

Comment:
please use Java convention for variables -> camel case e.g. clientId requestId etc.